src/Selenium2Library/keywords/element.py

```diff
@@ -217,6 +217,23 @@ def element_should_be_enabled(self, locator):
             raise AssertionError("Element '%s' is disabled." % (locator))
 
     @keyword
+    def element_should_be_focused(self, locator):
+        """Verifies that element identified with `locator` is focused.
+
+        Key attributes for arbitrary elements are `id` and `name`. See
+        `introduction` for details about locating elements.
+
+        New in SeleniumLibrary 2.0.1.
+        """
+        element = self.find_element(locator)
+        try:
+            focused = self.browser.execute_script('return document.activeElement;')
+        except (IOError, RuntimeError) as err:
+            raise AssertionError("Internal Error: '%s'" % (err))
```

Comment:
I have seen that proposal, but it is for Remote webdriver. It was not implemented, so the only alternative is to use JavaScript.